lib/bootloader_setup.pm

```diff
@@ -419,6 +419,8 @@ sub bootmenu_default_params {
         push @params, get_extra_boot_params();
     }
 
+    push @params, 'reboot_timeout=' . get_var('REBOOT_TIMEOUT', 0) unless (is_leap('<15.2') || is_sle('<15-SP2'));
```

Comment:
I see that you are a fan of using `unless` with multiple conditions =)
I would kindly ask you to rewrite it using `if`, especially as we can avoid negation by flipping versions.
https://www.perltutorial.org/perl-unless/#Perl_unless_statement_guidelines

Reply:
I could write it as

```perl
if (!is_leap('<15.2') && !is_sle('<15-SP2')) {
…
}
```

but I don't think that's better. If you were thinking of `is_leap('>=15.2') || is_sle('>=15-SP2')` then it should be obvious this is not equivalent, e.g. for Tumbleweed and others.
I prefer

```perl
do_this_default_action unless (old versions);
```

to steer the focus on the default action with the exceptions stated after that.